R/team_colour_palettes.R

```diff
@@ -0,0 +1,17 @@
+#' Returns a table with the colour palattes for all teams
+#'
+#' \code{get_afl_colour_palettes} returns a data frame containing the AFL team's primary, secondary and tertiary colours as applicable
+
+#' The data for this function is hosted on github.
+#'
+#' @return a data table containing team long name, team abbreviation, and colours
+#'
+#' \dontrun{
+#' # Gets all data
+#' get_afl_colour_palettes()
+#' }
+#' @importFrom here .here
+
+get_afl_colour_palettes <- function(){
+  return(readRDS(here::here("data-raw", "team_colours", "team_colours.rda")))
```

Comment:
For this one to work, we either need to host the file somewhere or include it as part of the package. Anything in `data-raw` doesn't get included when the package gets bundled so this function won't work. 

Generally I've been hosting things on a separate repo (see fitzroy_data) since internal package data has to be quite small. But this file is small enough I think we can just include it in the internal package data. 

If you put a script in `data-raw/team_colours` something like this is should work.

```
team_colours <- readRDS(here::here("data-raw", "team_colours", "team_colours.rda"))
usethis::use_data(team_colours)
```

We then need to document the `R/data.R` file as per the Documenting Data part of https://r-pkgs.org/data.html

Then - instead of needing a function, users can just reference this using `fitzRoy::team_colours`.

If you need, I can checkout your PR and edit this.

Reply:
Understood. I could host hte colour palette in the static files of my server.  I'll amend this later today. I have just pushed a commit to change the way the cached version is served to the user. I hope this suits better.

Reply:
Done.